awx/main/utils/external_logging.py

```diff
@@ -79,7 +79,7 @@ def construct_rsyslog_conf_template(settings=settings):
         parts.extend(['module(load="omhttp")', f'action({params})'])
     elif protocol and host and port:
         parts.append(
-            f'action(type="omfwd" target="{host}" port="{port}" protocol="{protocol}" action.resumeRetryCount="-1" template="awx")'  # noqa
+            f'action(type="omfwd" target="{host}" port="{port}" protocol="{protocol}" action.resumeRetryCount="-1" action.resumeInterval="{timeout}" template="awx")'  # noqa
```

Comment:
I wonder if we should set a max value, either on the setting enforced by django, or by setting this in the config:

```
action.resumeIntervalMax 3600
```

Reply:
Yea @rooftopcellist I don't think that's what that parameter is for; rsyslogd will progressively back off on retries, and `action.resumeIntervalMax` is the _max_ time it will raise the backoff.

Reply:
We *could* set some max value here, but it would probably mean a data migration (we'd need to make sure that nobody had *already saved* some value greater than the maximum we chose), and that's probably beyond the scope of this PR.